### PR TITLE
Enable Ubuntu tests in CI

### PR DESCRIPTION
**Pull request by baude**
Add updates required for ubuntu and run integration tests

Signed-off-by: Brent Baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by mheon** [comment]
LGTM once things go green

File: contrib/cirrus/setup_environment.sh [hunk 1]
```diff
@@ -39,6 +39,15 @@ done
 cd "${GOSRC}/"
 case "${OS_RELEASE_ID}" in
     ubuntu)
+	    apt-get update 
+	    if [[ "$OS_RELEASE_VER" == "19" ]]; then
+		    apt-get purge -y --auto-remove golang* 
+		    apt-get install -y golang-1.13 
+		    ln -s /usr/lib/go-1.13/bin/go /usr/bin/go
+	    fi
+	    if [[ "$OS_RELEASE_VER" == "20" ]]; then
```
Comment (TomSweeneyRedHat) [inline contrib/cirrus/setup_environment.sh:48]: should we do "> 19" here?

File: contrib/cirrus/setup_environment.sh [hunk 2]
```diff
@@ -39,6 +39,15 @@ done
 cd "${GOSRC}/"
 case "${OS_RELEASE_ID}" in
     ubuntu)
+	    apt-get update 
+	    if [[ "$OS_RELEASE_VER" == "19" ]]; then
+		    apt-get purge -y --auto-remove golang*
```
Comment (edsantiago) [inline contrib/cirrus/setup_environment.sh:44]: Could I recommend quoting the asterisk?
```bash
   ... --auto-remove "golang*"
```
Without the quotes, if there are one or more files named `golang<something>` in the current directory, the shell will expand that command line to `--auto-remove golang-file1 golang-file2 etc`. I realize this is unlikely and I'm probably being pedantic.

**Comment by rhatdan** [comment]
@mheon Did some of your pod handling code futher break Ubuntu?

**Comment by mheon** [comment]
Not that I'm aware of? That should all be OS-agnostic

**Comment by rhatdan** [comment]
@edsantiago Looks like your registry script is not working on ubuntu.

**Comment by edsantiago** [comment]
This is really strange. Everything points to a difference in the Ubuntu version of BATS: environment variables that are preserved in Fedora/RHEL are being clobbered in Ubuntu. The registry is starting, but subsequent tests aren't getting the right port / testuser / password. Is there some way to find out what version of BATS is being used? It's not listed in the `package_versions` CI step.

I'm also confused about why the `remote` tests are failing: this test is disabled on podman-remote. I think my test for `podman-remote` is failing because of the new APIv2 way. But that's another TODO for me.

**Comment by vrothberg** [comment]
Ubuntu:20.04 is shipping `Bats 1.2.0-dev` while F32 is on 1.1.0.

**Comment by vrothberg** [comment]
Opened https://github.com/containers/libpod/pull/6515. Bats v1.1.0 worked in the Ubuntu tests in the Google cloud.

**Pull request closed without merging**